x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -442,9 +427,11 @@ export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
   {
     id: SecurityPageName.administration,
     title: MANAGE,
-    path: MANAGE_PATH,
-    navLinkStatus: AppNavLinkStatus.hidden,
+    path: ENDPOINTS_PATH,
     features: [FEATURE.general],
+    navLinkStatus: AppNavLinkStatus.visible,
```

Comment:
Why is this set to `visible` now?

Reply:
this is the flag to make the `Manage` link to be visible in the global navigation

Reply:
Yeah, I understand it has to be visible but I don't understand why it wasn't visible before. Sorry for the noise, probably I'm missing something

Reply:
The `navLinkStatus` of the `AppDeepLink` is the global navigation visibility, If you open the global nav in main you'll see that _Manage_ is not present in the global nav before, _Endpoints_ is. This change removes `Endpoints` and adds `Manage`.